### PR TITLE
Explain return values from vector functions

### PR DESCRIPTION
**Pull request by JPryce-Aklundh**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1104.surge.sh

Updated pages:
- [Vector functions](https://neo4j-docs-cypher-1104.surge.sh/cypher-manual/5/functions/vector)

File: modules/ROOT/pages/functions/vector.adoc [hunk 1]
```diff
@@ -35,6 +35,8 @@ For more details, see the {link-vector-indexes}#similarity-functions[vector inde
 | Both vectors must be of the same dimension.
 | Both vectors must be {link-vector-indexes}#indexes-vector-similarity-cosine[*valid*] with respect to cosine similarity.
 | The implementation is identical to that of the latest available vector index provider (`vector-2.0`).
+| `vector.similarity.cosine()` returns the neighborhood of nodes along with their respective cosine similarity scores, sorted in descending order of similarity.
+The similarity score range from `0` and 1, with scores closer to `1` indicating a higher degree of similarity between the indexed vector and the query vector.
```
Comment (parnmatt) [inline modules/ROOT/pages/functions/vector.adoc]:
```suggestion
The similarity score range from `0` and `1`, with scores closer to `1` indicating a higher degree of similarity between the indexed vector and the query vector.
```

File: modules/ROOT/pages/functions/vector.adoc [hunk 2]
```diff
@@ -63,6 +65,8 @@ For more details, see the {link-vector-indexes}#similarity-functions[vector inde
 | Both vectors must be of the same dimension.
 | Both vectors must be {link-vector-indexes}#indexes-vector-similarity-euclidean[*valid*] with respect to Euclidean similarity.
 | The implementation is identical to that of the latest available vector index provider (`vector-2.0`).
+| `vector.similarity.euclidean()` returns the neighborhood of nodes along with their respective Euclidean similarity scores, sorted in descending order of similarity.
+The similarity score range from `0` and 1, with scores closer to `1` indicating a higher degree of similarity between the indexed vector and the query vector.
```
Comment (parnmatt) [inline modules/ROOT/pages/functions/vector.adoc]:
```suggestion
The similarity score range from `0` and `1`, with scores closer to `1` indicating a higher degree of similarity between the indexed vector and the query vector.
```

**Review by parnmatt — APPROVED** [review]
Looks reasonable, just suggesting being consistent with the numbers being in backticks

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**